projects/batfish-client/src/main/java/org/batfish/client/BfCoordWorkHelper.java

```diff
@@ -958,9 +958,11 @@ public JSONObject listAnalyses(String containerName) {
 
          String[] questionList = new String[questions.length()];
 
+         @SuppressWarnings("unchecked")
```

Comment:
I don't quite understand the argument here. The library was written without any generics, but in practice all JSON keys are strings, and the library bears this out.

What's the problem with suppressing a cast you know is correct?

(There is no cast later FWICT.)

I did add a comment to the cast, and also simplified the code even further using `Iterators` library call.